app/routes/TaskResource.js

```diff
@@ -0,0 +1,149 @@
+    *                     profile_picture: user profile picture { type: "Image", url: picture url }
+    *                   }
+    *                 } , ... ]
+    *      },
+    * 	errors:
+    * 	[
+    * 	  {
+    * 	     param: the field that caused the error,
+    * 	     value: the value that was provided for that field,
+    * 	     type: the type of error that was caused ['required', 'validity', 'unique violation']
+    * 	  }, {...}, ...
+    * 	]
+    * }
+    */
+    app.get('/api/task/:id', auth, TaskController.show);
+
+    /**
+    * A POST route responsible for creating a task.
+    * @var /api/task POST
+    * @name /api/task POST
+    * @example The user requesting the route has to be of type 'High Board' at least.
+    * @example the route expects the access token as 'Authorization' and the user agent as 'user_agent' in the request headers with one of the following values ['Web', 'IOS', 'Android']
+    * @example The route expects a body Object in the following format
+    * {
+    *     title: the task's title [required],
+    *     description: the task's description String [optional|null],
+    *     deadline: the task's end date and time "YYYY-MM-DD HH:MM:SS" [required],
+    *     priority: the task's priority in [1, 3, 5, 8][required],
+    *     evaluation: task's evaluation [optional|null],
+    *     assigned_to: The task's assignees Array of integers user ids [optional|[]]
+    * }
+    * @example The route returns as a response an object in the following format
+    * {
+    *   status: succeeded/failed,
+    *   message: String showing a descriptive text,
+    *   errors:
+    *   [
+    *     {
+    *         param: the field that caused the error,
+    * 	     value: the value that was provided for that field,
+    * 	     type: the type of error that was caused ['required', 'validity', 'unique violation']
+    * 	 }, {...}, ...
+    *   ]
+    * }
+    */
+    app.post('/api/task', auth, high, TaskController.store);
+
+    /**
+    * A PUT route responsible for updating a task.
+    * @var /api/task/{id} PUT
+    * @name /api/task/{id} PUT
+    * @example The user requesting the route has to be of type 'High Board' at least.
+    * @example The route expects a task id for the desired task in the URL in replace of '{id}'
+    * @example the route expects the access token as 'Authorization' and the user agent as 'user_agent' in the request headers with one of the following values ['Web', 'IOS', 'Android']
+    * @example The route expects a body Object in the following format
+    * {
+    *     title: the task's title [optional],
+    *     description: the task's description String [optional|null],
+    *     deadline: the task's end date and time "YYYY-MM-DD HH:MM:SS" [optional],
+    *     priority: the task's priority in [1, 3, 5, 8][optional],
+    *     status: the task's status in ["New", "Done", "In Progress", "Ready"][optional],
+    *     evaluation: task's evaluation [optional|null],
```

Comment:
in the update there are no default values as if you didn't receive any of them in the body it should stay unedited in the database